next/src/components/layouts/Sections.tsx

```diff
@@ -33,7 +32,14 @@ import TootootEventsSection from '@/src/components/sections/TootootEventsSection
 import VideosSection from '@/src/components/sections/VideosSection'
 import { SectionsFragment } from '@/src/services/graphql'
 
-const SectionContent = ({ section }: { section: SectionsFragment }) => {
+type Section = SectionsFragment
```

Comment:
Hmm why do we have to specify this type? Could we use just `SectionsFragment` directly?